bin/varnishd/cache/cache_vcl.c

```diff
@@ -257,9 +257,23 @@ VCL_Panic(struct vsb *vsb, const char *nm, const struct vcl *vcl)
 void
 vcl_get(struct vcl **vcc, struct vcl *vcl)
 {
+	struct vcl *old;
+
 	AN(vcc);
 
+	old = *vcc;
+	*vcc = NULL;
+
+	if (old != NULL)
+		CHECK_OBJ(old, VCL_MAGIC);
```

Comment:
use CHECK_OBJORNULL()

Reply:
right, thank you. The `if`-block used to contain more, this was a residue of a previous incarnation